src/lib/components/Mission.svelte

```diff
@@ -1,125 +1,106 @@
-<script>
+<script lang="ts">
+	import { page } from '$app/stores';
 	import SectionHeader from './sections/SectionHeader.svelte';
+	let activeTab = 'Mission'
+
+  const mission = [
```

Comment:
mission and vision is unique
you can just use const mission = {...mission data here}, the same as mission